pkg/instancetype/instancetype.go

```diff
@@ -92,6 +92,17 @@ func CreateControllerRevision(vm *virtv1.VirtualMachine, object runtime.Object)
 	}, nil
 }
 
+func ensureTypeMeta(obj runtime.Object) error {
```

Comment:
This should really live somewhere more generic but that's something for a followup.

Reply:
I dropped it for now as it is no longer needed inside this series.